sw/device/lib/runtime/otbn.c

```diff
@@ -190,3 +190,24 @@ otbn_result_t otbn_dump_dmem(const otbn_t *ctx, uint32_t max_addr) {
 
   return kOtbnOk;
 }
+
+otbn_result_t otbn_zero_instruction_memory(otbn_t *ctx) {
+  if (ctx == NULL) {
+    return kOtbnBadArg;
+  }
+
+  size_t imem_size_words =
+      dif_otbn_get_imem_size_bytes(&ctx->dif) / sizeof(uint32_t);
+  bool retval = kOtbnOk;
+  for (size_t i = 0; i < imem_size_words; ++i) {
+    const uint32_t zero = 0;
```

Comment:
```suggestion
    const uint32_t kZero = 0;
```

Reply:
Same here, this has just been copied over but got removed now.